### PR TITLE
#4 redesign settings page2

### PR DESCRIPTION
**Pull request by JoeHorvath**
Deleted password controller and html and moved to settings. Added system to display yellow edge border on fields in settings that have been edited and not saved.

### CODE REVIEW

**Review by oliver-allen — APPROVED** [review]
Looks alright to me

File: src/main/webapp/app/account/settings/settings.component.ts [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+    password: string;
+    currentPassword: string;
+    confirmPassword: string;
+    wrongPassword: boolean;
+
     constructor(
-        private account: AccountService,
-        private principal: Principal
+        private accountService: AccountService,
+        private principal: Principal,
```
Comment (xavier630) [inline src/main/webapp/app/account/settings/settings.component.ts]: What's a principal?

**Review by xavier630 — APPROVED** [review]
Looking good. Nice work!

**Comment by softeng-701** [comment]
@xavier630 Squash please!

File: src/main/webapp/app/account/settings/settings.component.ts (hunk 1 above)
Comment (JoeHorvath) [reply src/main/webapp/app/account/settings/settings.component.ts]: I don't know, it was already in settings.component. Looks like its maybe used to identify the user.

**Comment by JoeHorvath** [comment]
@kblincoe  I have squashed the commits as much as I can but the merges from master do not appear to be squashable.

**Comment by kblincoe** [comment]
@JoeHorvath only squash your actual commits, don't worry about merge commits

**Comment by softeng-701** [comment]
@JoeHorvath can you squash please?

**Comment by JoeHorvath** [comment]
@softeng-701 I have squashed all the non merge commits already as instructed. Cheers

**Pull request merged**